src/machine/board_pca10056.go

```diff
@@ -28,6 +28,12 @@ const (
 	UART_RX_PIN Pin = 8
 )
 
+// UART0 is the NRF UART and UART1 is the nRF52840's USB device
+var (
+	UART0 = NRF_UART0
+	UART1 = USB
```

Comment:
The nrf52840 has an UART1 (actually UARTE1) so this would conflict.
I think it would be better to leave `UART1` undefined and expect programs to use `machine.USB` if they specifically need to write to that output.
(The same is true for other boards that primarily use a real UART for output).

Reply:
Should I also leave `UART1` undefined for boards that will use `USB` as the primary output? (programs would have to access `NRF_UART0` if they want to want to use the real UART)

Reply:
> Should I also leave `UART1` undefined for boards that will use `USB` as the primary output? (programs would have to access `NRF_UART0` if they want to want to use the real UART)

Not sure. @deadprogram any opinion on this?

Reply:
So up until now, we have typically treated `UART0` as either a "real" UART or the USB-CDC interface, depending on what the board supports.

However, that now seems a little clumsy. I think we should make a chance and instead do something like what we do with `machine.LED` to have a well-known alias. We could name it something like `machine.SERIAL` or `machine.SERIALPORT`.

- Boards where there is a USB-CDC like basically all of the Adafruit boards, `machine.SERIAL = machine.USB`.
- Boards where there the "default" is actually a UART perhaps via a debug chip. then `machine.SERIAL = machine.UART0`.

If the user code needs to communicate with a specific UART, then that is easily handled. The default is the "sensible" option. And we can define all of the available ports per board without having to leave something out, or renumber.

Reply:
This sounds very reasonable. Completely agree.

That would also fix weird cases like this:

https://github.com/tinygo-org/tinygo/blob/1d913a62bc0ceef841953058c2c6fb1dd2a40f6a/src/machine/board_bluepill.go#L62-L68

Reply:
`machine.Serial` seems good. Also sounds good to make those changes in a separate PR.